climada/entity/exposures/nightlight.py

```diff
@@ -363,7 +364,7 @@ def load_nightlight_noaa(ref_year=2013, sat_name=None):
             for pre_i in np.arange(ini_pre, end_pre, -1):
                 url = NOAA_SITE + 'F' + str(pre_i) + str(ref_year) + '.v4.tar'
                 try:
-                    file_down = download_file(url)
+                    file_down = download_file(url, download_dir=SYSTEM_DIR)
```

Comment:
That's what I thought too.. but it was not the case for me when I ran it, it was saving the downloaded files in my current directory

Reply:
Maybe @emanuel-schmid can comment on this?

I'm not sure what exactly is happening here, but the rest of this PR is fine from my side.

Reply:
yep, same from my side, all good, but would be nice if Emanuel could explain / help out on this one.

Reply:
The parameter `download_dir` of the `download_file` functions was introduced with the changes of the config settings.
(Before the changes, it used to download files into the working directory. Which was sometimes countered by undesired `os.chdir()` calls.)
The function's default directory is `CONFIG.local_data.save_dir`. The configuration's default value for this is `./results`.

`SYSTEM_DIR` is a constant, but it's not exactly hard coded. Its value is defined as `CONFIG.local_data.system`.

I hope these explanations help clarifying the behavior?

Reply:
> SYSTEM_DIR is a constant, but it's not exactly hard coded. Its value is defined as CONFIG.local_data.system.

Thanks! I think that is the answer to @Evelyn-M's original question and solves this confusion.

Reply:
thanks Emanuel, yes, it was creating a ./results folder in the working directory and saving files therein. Problem is, the loading function called after was expecting the files to be in SYSTEM_DIR, so that was the problem.

So maybe that also explaines why the same did not crash on Jenkins? (see #143) Can it be that there it is CONFIG.local_data.save_dir == CONFIG.local_data.system ? (my guess)